### PR TITLE
[JSONAPI ] Add name convention in filtering

### PR DESCRIPTION
**Pull request by patricia-martinez-imatia**

### CODE REVIEW

File: projects/ontimize-web-ngx/src/lib/util/util.ts [hunk 1]
```diff
@@ -583,54 +589,36 @@ export class Util {
...
-    return Util.mapKeys(data, (val, key) => convention === 'lower' ? Util.toLowerCase(key) : Util.toUpperCase(key));
-  }
-
   /**
    * Map keys of object
    * For example: converting the keys of an object to uppercase
    */
   static readonly mapKeys = (obj, fn) =>
     Object.keys(obj).reduce((acc, k) => {
+      console.log(obj[k], acc);
```
Comment (daniel-grana) [inline projects/ontimize-web-ngx/src/lib/util/util.ts]: Remove this console.log

File: projects/ontimize-web-ngx/src/lib/util/util.ts [hunk 2]
```diff
@@ -583,54 +589,36 @@ export class Util {
...
       acc[fn(obj[k], k, obj)] = obj[k];
+      console.log(obj[k],k, acc);
```
Comment (daniel-grana) [inline projects/ontimize-web-ngx/src/lib/util/util.ts]: Remove this console.log

File: projects/ontimize-web-ngx/src/lib/services/ontimize/ontimize-ee.service.ts [hunk 3]
```diff
@@ -65,6 +66,10 @@ export class OntimizeEEService extends OntimizeBaseService implements IDataServi
     av = (Util.isDefined(av)) ? av : this.av;
     sqltypes = (Util.isDefined(sqltypes)) ? sqltypes : this.sqltypes;
 
+    kv = NameConvention.parseDataToNameConvention(this._appConfig.nameConvention, kv);
```
Comment (daniel-grana) [inline projects/ontimize-web-ngx/src/lib/services/ontimize/ontimize-ee.service.ts:69]: What happens if an application does not define 'nameConvention' on its appConfig? I do not see any initialization of this parameter

**Review by daniel-grana — CHANGES_REQUESTED** [review]

File: projects/ontimize-web-ngx/src/lib/services/ontimize/ontimize-ee.service.ts (hunk 3 above)
Comment (patricia-martinez-imatia) [reply projects/ontimize-web-ngx/src/lib/services/ontimize/ontimize-ee.service.ts:69]: It is initializate in [app.config.ts](https://github.com/OntimizeWeb/ontimize-web-ngx/blob/feature/15.x.x-jsonapi-improve-name-convention/projects/ontimize-web-ngx/src/lib/config/app-config.ts), line 115
` getNameConvention() {
    return this._config.nameConvention || 'database';
  }`

**Review by daniel-grana — APPROVED** [review]

**Pull request merged**